Sources/Danger/GitHubDSL.swift

```diff
@@ -17,489 +13,449 @@ public struct GitHub: Decodable {
-    // MARK: - CodingKeys
-
-    enum CodingKeys: String, CodingKey {
-        case id
-        case name
-        case owner
-        case description
-        case fullName = "full_name"
-        case isPrivate = "private"
-        case isFork = "fork"
-        case htmlURL = "html_url"
+        /// The list of teams of whom a review has been requested.
+        public let teams: [Team]
     }
-
-    // MARK: - Properties
-
-    /// Generic UUID.
-    public let id: Int
-
-    /// The name of the repo, e.g. "danger-swift".
-    public let name: String
-
-    /// The full name of the owner + repo, e.g. "Danger/danger-swift"
-    public let fullName: String
-
-    /// The owner of the repo.
-    public let owner: GitHubUser
-
-    /// A boolean stating whether the repo is publicly accessible.
-    public let isPrivate: Bool
-
-    /// A textual description of the repo.
-    public let description: String?
-
-    /// A boolean stating whether the repo is a fork.
-    public let isFork: Bool
-
-    /// The root web URL for the repo, e.g. https://github.com/artsy/emission
-    public let htmlURL: String
 }
 
-// MARK: - GitHubReview
+extension GitHub {
+    /// Represents 'head' in PR
+    public struct MergeRef: Decodable, Equatable {
+        /// The human display name for the merge reference, e.g. "artsy:master".
+        public let label: String
 
-public struct GitHubReview: Decodable, Equatable {
-    // MARK: - CodingKeys
+        /// The reference point for the merge, e.g. "master"
+        public let ref: String
 
-    enum CodingKeys: String, CodingKey {
-        case user
-        case id
-        case body
-        case state
-        case commitId = "commit_id"
-    }
+        /// The reference point for the merge, e.g. "704dc55988c6996f69b6873c2424be7d1de67bbe"
+        public let sha: String
 
-    // MARK: - ReviewState
+        /// The user that owns the merge reference e.g. "artsy"
+        public let user: User
 
-    public enum ReviewState: String, Decodable {
-        case approved = "APPROVED"
-        case requestedChanges = "CHANGES_REQUESTED"
-        case comment = "COMMENTED"
-        case pending = "PENDING"
-        case dismissed = "DISMISSED"
+        /// The repo from which the reference comes from
+        public let repo: Repo
     }
-
-    /// The user who has completed the review or has been requested to review.
-    public let user: GitHubUser
-
-    /// The id for the review (if a review was made).
-    public let id: Int?
-
-    /// The body of the review (if a review was made).
-    public let body: String?
-
-    /// The commit ID the review was made on (if a review was made).
-    public let commitId: String?
-
-    /// The state of the review (if a review was made).
-    public let state: ReviewState?
 }
 
-// MARK: - GitHubCommit
+extension GitHub {
+    public struct Repo: Decodable, Equatable {
+        enum CodingKeys: String, CodingKey {
+            case id
```

Comment:
<!--
  0 failure: 
  1 warning:  Enum element name...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/GitHubDSL.swift;
  Line: 206;
-->

- :warning: Enum element name should be between 3 and 40 characters long: 'id' (`identifier_name`)